tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -288,3 +303,246 @@ func testHNSNetworksCreated(t *testing.T) {
 	assert.Contains(t, stdoutString, "Name                   : OpenShiftNetwork",
 		"Could not find OpenShiftNetwork in list of HNS Networks")
 }
+
+// testEastWestNetworking deploys Windows and Linux pods, and tests that the pods can communicate
+func testEastWestNetworking(t *testing.T) {
+	// Preload the image that will be used on the Windows node, to prevent download timeouts
+	// and separate possible failure conditions into multiple operations
+	err := pullDockerImage(windowsServerImage)
+	require.NoError(t, err, "Could not pull Windows Server image")
+
+	// This will run a Server on the container, which can be reached with a GET request
+	winServerCommand := []string{"powershell.exe", "-command",
+		"<#code used from https://gist.github.com/wagnerandrade/5424431#> ; $$listener = " +
```

Comment:
Any specific reason for using this command espcially reference to `wagnerandrade..`?

Reply:
This command was what @aravindhp used and had hosted for the kubecon demo

Reply:
That shouldn't be the reason to use that here. What we did for kubecon was hack something together just to get it to work.